demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -900,11 +900,16 @@ def is_valid_fetch(self) -> bool:
         """
         fetch_params_exist = True
         if self.current_file.get('script', {}).get('isfetch') is True:
+
+            # get the iten marketplaces to decide which are the required params
+            marketplaces = get_item_marketplaces(item_path=self.file_path, item_data=self.current_file)
+            is_xsoar_marketplace = not marketplaces or MarketplaceVersions.XSOAR.value in marketplaces
```

Comment:
Please add a comment explaining that if xsoar is in the item's marketplaces, the validation will be according to xsoar format

Reply:
done